tests/test_symbol.py

```diff
@@ -145,9 +150,13 @@ def test_by_name_precedence(self):
         expected = 0xFFFF0008
```

Comment:
You should probably add a variable for `expected - 0x8` and use that in the assertion and when creating `lower`. Maybe just `other = expected - 0x8`?